### PR TITLE
curves tab

### PR DESCRIPTION
**Pull request by kunchenguid**
image

### CODE REVIEW

File: packages/shared/src/components/CombatReport/CombatCurves/CurveChart.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+import { Legend, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
```
Comment (prsauer) [inline packages/shared/src/components/CombatReport/CombatCurves/CurveChart.tsx:2]: Have you used recharts a lot? I've been using https://formidable.com/open-source/victory/docs/victory-chart/ in other repos to decent results

File: packages/shared/src/components/CombatReport/index.tsx [hunk 2]
```diff
@@ -83,21 +84,14 @@ export const CombatReport = ({ combat, anon }: IProps) => {
             Replay
           </a>
         </div>
-        {activeTab === 'summary' && (
-          <div className="mt-4 mx-2">
-            <CombatSummary />
+        <div className="mt-4 mx-2 flex-1 relative overflow-y-scroll">
```
Comment (prsauer) [inline packages/shared/src/components/CombatReport/index.tsx:87]: using mx-2 and padding from the outer container is causing the scrollbar to be offset from the edge of the screen
ultimately not a deal breaker for me but something to consider

image

File: packages/desktop/components/DesktopLayout/index.tsx [hunk 3]
```diff
@@ -100,7 +100,7 @@ export const DesktopLayout = !window.wowarenalogs
           >
             <AuthProvider>
               <LocalCombatsContextProvider>
-                <div className="w-screen h-screen flex flex-col bg-base-300">
+                <div className="w-screen h-screen flex flex-col bg-base-300 overflow-hidden">
```
Comment (prsauer) [inline packages/desktop/components/DesktopLayout/index.tsx:103]: Not sure this resolved it entirely for me -- still seeing horizontal scroll on the Summary page

image

File: packages/shared/src/components/CombatReport/CombatCurves/CurveChart.tsx (hunk 1 above)
Comment (kunchenguid) [reply packages/shared/src/components/CombatReport/CombatCurves/CurveChart.tsx:2]: yeah I've used recharts in other projects and it's pretty good. any particular feature in victory chart that you think would be useful for us? I can check if they are available through recharts.

File: packages/shared/src/components/CombatReport/index.tsx (hunk 2 above)
Comment (kunchenguid) [reply packages/shared/src/components/CombatReport/index.tsx:87]: oh yes the margin needs to be removed.

**Review by prsauer — APPROVED** [review]

**Pull request merged**